pkg/features/kube_features.go

```diff
@@ -325,6 +325,14 @@ const (
 	// beta:  v1.18
 	// GA:    v1.20
 	//
+	// Enables the endPort field in NetworkPolicy to enable a Port Range behavior in Network Policies.
+	NetworkPolicyEndPort featuregate.Feature = "NetworkPolicyEndPort"
+
+	// owner: @rikatz
+	// alpha: v1.20
```

Comment:
Generally we only comment feature states that have already merged, so this should only include `alpha` for this PR